src/stylesheets/ToggleSwitch.scss

```diff
@@ -0,0 +1,38 @@
+@import "colors";
+
+
+.wrapper {
+  padding: 50px;
+}
+
+.toggle-component {
+  font-size: 2rem;
+  color: $light-gray;
+  width: 50px;
+  height: 25px;
+  background: #777777;
+  border: 1px solid $silver-gray;
+  position: relative;
+  border-radius: 13px;
+  cursor: pointer;
+  transition: border-color background-color 0.3s;
+  margin: 10px;
```

Comment:
Could we just to top and bottom margin here? Like `margin: 10px 0;`
I think the left margin is making the toggle not align with others in the popup.
![image](https://user-images.githubusercontent.com/33335963/83802350-6f70f880-a65f-11ea-8331-447ea7f17dab.png)